### PR TITLE
docs: add docs for #2482

### PR DESCRIPTION
**Pull request by fubuloubu**
fixes: #2400

### CODE REVIEW

File: docs/built-in-functions.rst [hunk 1]
```diff
@@ -143,6 +146,15 @@ Chain Interaction
         def foo(_target: address) -> Bytes[32]:
             response: Bytes[32] = raw_call(_target, 0xa9059cbb, max_outsize=32, value=msg.value)
             return response
+            
+        @external
+        @payable
+        def bar(_target: address) -> Bytes[32]:
+            success: bool = True
```
Comment (charles-cooper) [inline docs/built-in-functions.rst:153]: I would set this to `False` as a general practice

File: docs/built-in-functions.rst [hunk 2]
```diff
@@ -143,6 +146,15 @@ Chain Interaction
...
+            response: Bytes[32] = b""
+            success, response = raw_call(_target, 0xa9059cbb, max_outsize=32, value=msg.value)
```
Comment (charles-cooper) [inline docs/built-in-functions.rst:155]: needs `revert_on_failure` kwarg

**Review by charles-cooper — CHANGES_REQUESTED** [review]
example needs `revert_on_failure` kwarg

File: docs/built-in-functions.rst (hunk 1 above)
Comment (fubuloubu) [reply docs/built-in-functions.rst:153]:
```suggestion
            success: bool = False
```

File: docs/built-in-functions.rst (hunk 2 above)
Comment (fubuloubu) [reply docs/built-in-functions.rst:155]:
```suggestion
            success, response = raw_call(_target, 0xa9059cbb, max_outsize=32, value=msg.value, revert_on_failure=False)
```

**Review by charles-cooper — APPROVED** [review]

**Pull request merged**